recipe/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D BUILD_SHARED_LIBS=ON ^
+      -D ENABLE_TESTS=OFF ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D ZLIB_LIBRARY=%LIBRARY_LIB%\zlib.lib ^
```

Comment:
The `.lib`? I thought in Windows `.lib` implied static and `.dll` implied dynamic, much as `.so` and `.a` on Linux for dynamic/static?